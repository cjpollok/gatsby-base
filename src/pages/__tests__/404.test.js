import React from 'react';
import { mount } from 'enzyme';

import NotFoundPage, { NotFoundPageBase } from '../404';

describe('404 Component', () => {
  it('renders correctly', () => {
    const wrapper = mount(<NotFoundPage />);
    const base = wrapper.find(NotFoundPageBase);
    expect(base.render()).toMatchSnapshot();
  });
});
