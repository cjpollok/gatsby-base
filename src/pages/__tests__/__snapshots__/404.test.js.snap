// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Component renders correctly 1`] = `
<p>
  You just hit a route that doesn't exist.
</p>
`;
